docs/developers-clear-containers-install.md

```diff
@@ -36,8 +36,9 @@ Since the installation guide will have installed packaged versions of
 all required components, it is only necessary to install the source for
 the component(s) you wish to develop with.
 
-**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers).
-Both projects ship ``cc-proxy`` and they are not compatible with each other.
+**IMPORTANT:** Do not combine [Clear Containers 2.x](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.x](https://github.com/clearcontainers) on the same system.
+Both projects ship ``cc-proxy`` and ``cc-shim`` which are not compatible with each other.
```

Comment:
Is there a reason we go back and fourth between double and singe back-ticks for literal text?
I am seeing that the majority of new docs coming through are using single `, this doc also uses single earlier in the text (line 25).

Reply:
This edit was just an update to an existing doc (containing the double-ticks). However, I agree so will push a new commit with single ticks...

Reply:
Done.